browser-interface/build.js

```diff
@@ -168,12 +168,7 @@ function createWorker(entry, outfile) {
 }
 
 async function compileJs() {
-  ensureFileExists(DIST_PATH, 'unity.loader.js')
-  const injectUnityPath = path.resolve(DIST_PATH, 'unity.loader.js')
-
-  for (let file of Object.values(generatedFiles)) {
-    ensureFileExists(DIST_PATH, file)
-  }
```

Comment:
Every check has a story...

I'd keep the ensureFileExists, as a sanity check that the build finished succesfully

Reply:
Nevermind, just saw it was moved to the next file